src/main/java/at/hannibal2/skyhanni/features/garden/CropType.kt

```diff
@@ -12,28 +12,39 @@ enum class CropType(
     val specialDropType: String,
     val baseDrops: Double,
     iconSupplier: () -> ItemStack,
+    val simpleName: String,
```

Comment:
remove

Reply:
If I remove this you can't tab complete the crops for the command

Reply:
Didn't notice tsb compete exists. In this case maybe use the same logic as for gfs tab complete

Reply:
I'm still more a fan of my existing method but if you really want it changed i can